src/App.js

```diff
@@ -1,41 +1,14 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import { Home } from "./Pages/Index";
+import { Route, Routes } from "react-router-dom";
+import { useTheme } from "./Contexts/Index";
```

Comment:
Use index.js instead of Index.js in the Contexts folder, this is the recommended approach and the import statement will be shortened to-
 import { useTheme } from "./Contexts